src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -7,6 +7,7 @@
 use Illuminate\Support\Arr;
 use Illuminate\Support\Str;
 use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\Crypt;
```

Comment:
Not sure this cross dependency is a good idea.

Reply:
Any suggestions how encryption can be done without it, short of actually duplicating the encryption code? Using the standard Laravel encryption means you can use raw queries easily, if needed. Using custom encryption for this use case is messy.

Is there a way to inject a typehinted contract into the model automatically? That might keep it less dependant. 🤔

Reply:
I am really saying the whole idea of eloquent knowing that encryption exists is a bad idea, so I';m :-1: on this PR.